mix.exs

```diff
@@ -31,9 +31,8 @@ defmodule Samly.Mixfile do
     [
       {:plug, "~> 1.6"},
       {:esaml, "~> 4.2"},
-      {:sweet_xml, "~> 0.6.6"},
-      {:ex_doc, "~> 0.19.0", only: :dev, runtime: false},
-      {:inch_ex, "~> 1.0", only: [:dev, :test]}
+      {:sweet_xml, "~> 0.6"},
```

Comment:
This is the only substantive change -- allows dependents to use newer minor versions of sweet_xml.